src/tests/unit-tests.cpp

```diff
@@ -1,6 +1,64 @@
+#include "../app/doctor/doctor.hpp"
```

Comment:
I believe we can remove this includes with linking libraries into the test target (please refer to my comment for src/tests/CMakeLists.txt). It would at the very list remove need for nasty paths fixed.

Even if not, they should be specified with target_include_directories() to avoid fixing the paths to includes.